### DISCUSSION TITLE
Uboss

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#665] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Uboss is gonna have its on repo soon, and also have more features, let me know some below.

**Comment by Hilololsusy** [comment, marked as answer]
ok.

**Comment by Aka-but-nice** [comment]
> Uboss is gonna have its on repo soon, and also have more features, let me know some below.

I can't wait until Uboss gets its on repository
